__tests__/app.test.js

```diff
@@ -83,4 +83,45 @@ describe("API's", () => {
         });
     });
   });
+  describe("/api/articles", () => {
+    test("GET 200: Gets all articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(13);
```

Comment:
Great that you've included a length check of the array to avoid false positives if you get an empty array 👍